src/Octokit.Webhooks/Models/DeploymentEvent/DeploymentCheckRunStatus.cs

```diff
@@ -0,0 +1,20 @@
+namespace Octokit.Webhooks.Models.DeploymentEvent
+{
+    using System.Runtime.Serialization;
+    using System.Text.Json.Serialization;
+
+    [JsonConverter(typeof(JsonStringEnumMemberConverter))]
+    public enum DeploymentCheckRunStatus
```

Comment:
Schema reference[^1] plus the values from #52

[^1]: https://github.com/octokit/webhooks/blob/d93eebf4c48dc3c36f9c7ceffab356fa444bf9fc/payload-types/schema.d.ts#L2214